testsuite/openshift/objects/__init__.py

```diff
@@ -46,4 +46,6 @@ def commit(self):
 
     def delete(self, ignore_not_found=True, cmd_args=None):
         """Deletes the resource, by default ignored not found"""
-        return super().delete(ignore_not_found, cmd_args)
+        deleted = super().delete(ignore_not_found, cmd_args)
+        self.committed = False
```

Comment:
I'll add a check, but I don't think it is necessary here, since the function will raise an error on incorrect deletion and stop the program execution either way.